examples/redis_example/README.md

```diff
@@ -0,0 +1,20 @@
+This example shows how to set up the Python tracker with a Redis database and a Redis worker to forward events to a Snowplow pipeline.
+
+#### Installation
+- Install the Python dependencies for this project.
```

Comment:
Before this, could we add another step to run this in the root folder of the project:

```
python setup.py install
```

That will install the tracker package so that the example can use it.